src/App.tsx

```diff
@@ -1,27 +1,52 @@
 import React from 'react';
+import { Clock } from './components/clock/Clock';
+
 import './App.scss';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+interface State {
+  isClockVisible: boolean,
+  clockName: number,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    isClockVisible: true,
+    clockName: 1,
+  };
+
+  render(): React.ReactNode {
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <div className="clock-box">
+          <p className="clock-box__contetn">Current time:</p>
+          {this.state.isClockVisible && (
```

Comment:
You can use destructuring for state property and leave just {isClockVisible ... }.

Reply:
agree, add destructuring for state

Reply:
Already added)